cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -281,6 +351,28 @@ func buildPartitionList(ptable *sfdiskPartitionTable, pv *gadget.LaidOutVolume)
 	return buf, toBeCreated
 }
 
+// listCreatedPartitions returns a list of partitions created during the
+// install process.
+// TODO:UC20: also provide a mechanism for MBR (RPi)
+func listCreatedPartitions(dl *DeviceLayout) []string {
+	created := make([]string, 0, len(dl.partitionTable.Partitions))
+	for _, p := range dl.partitionTable.Partitions {
+		if !strutil.ListContains(createdPartitionGUID, strings.ToUpper(p.Type)) {
```

Comment:
we do this check in two places now, so we might want to make a small helper for it:

```
// creationSupported returns whether we support and expect to create partitions of the given type, it also means
// we are ready to remove them for re-installation or retried installation if they are appropriately marked with
// createdPartitionAttr.
func creationSupported(ptype string) bool ...
```

for this,  can be done in a follow up